tests/gold_tests/h2/h2bigclient.py

```diff
@@ -0,0 +1,81 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import json
+from hyper import HTTPConnection
+import hyper
+import argparse
+
+def getResponseString(response):
+    typestr = str(type(response))
+    if typestr.find('HTTP20') != -1:
+        string = "HTTP/2 {0}\r\n".format(response.status)
+    else:
+        string = "HTTP {0}\r\n".format(response.status)
+    string+='date: '+response.headers.get('date')[0].decode('utf-8')+"\r\n"
+    string+='server: '+response.headers.get('Server')[0].decode('utf-8')+"\r\n"
+    return string
+
+def makerequest(port):
+    hyper.tls._context = hyper.tls.init_context()
+    hyper.tls._context.check_hostname = False
+    hyper.tls._context.verify_mode = hyper.compat.ssl.CERT_NONE
+
+    conn = HTTPConnection('localhost:{0}'.format(port), secure=True)
```

Comment:
This reminded me, I need to pass address as a parameter to makerequest (h2client.py), rather than having HTTPConnection connect to localhost all the time.